shopno/index.html

```diff
@@ -0,0 +1,586 @@
+                        </div>
+                    </li>
+                </ul>
+            </div>
+        </section>
+        <section class="contact">
+            <h2 style="display: none;">Our contact</h2>
+            <div class="contact__container container">
+                <form id="form" action="#" class="contact__form contact__block contact-form">
+                    <input type="text" class="contact-form__input contact__text" placeholder="Name*" required>
+                    <input type="text" class="contact-form__input contact__text" placeholder="Subject*" required>
+                    <input type="email" class="contact-form__input contact__text" placeholder="E-mail*" required>
+                    <textarea class="contact-form__input contact-form__input--big contact__text" placeholder="Message*" required></textarea>
+                    <button class="contact-form__button">Send message</button>
+                </form>
+                <div class="contact__block contact__block2 contact-block2"  id="contact-section">
+                    <div class="contact-block2__text-wrap">
+                        <p class="contact-block2__text contact__text">Adress line 1</p>
+                        <p class="contact-block2__text contact__text">Adress line 2</p>
+                        <p class="contact-block2__text contact__text">Phone: <a  class="contact__text" href="tel:+548945645">+548945645</a></p>
+                        <p class="contact-block2__text contact__text">Fax: <a  class="contact__text" href="fax:+45645654546">+45645654546</a></p>
+                        <br>
+                        <p class="contact-block2__text contact__text">
+                            Web: <a class="contact__text" href="#">www.designagency.net</a>
+                        </p>
+                        <p class="contact-block2__text contact__text">
+                            E-mail: <a class="contact__text" href="#">info@designagency.net</a>
+                        </p>
+                    </div>
+                    <iframe class="contact-block2__map" src="https://www.google.com/maps/embed?pb=!1m14!1m12!1m3!1d2796.0695556156547!2d27.568824063326627!3d53.89256257143246!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!5e0!3m2!1sru!2sby!4v1633363958576!5m2!1sru!2sby" width="400" height="300" style="border:0;" allowfullscreen="" loading="lazy"></iframe>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer class="footer">
+        <h2 class="footer__title">
+            Follow us
+        </h2>
+        <ul class="footer__social social">
+            <li class="social__item">
+                <a href="#" class="social__link">
+                    <picture>
+                        <source 
+                            srcset="./src/images/pinterest.webp"
+                            type="image/webp">
+                        <img class="social__icon" src="./src/images/pinterest.png" alt="Social link">
```

Comment:
You forgot your favorite `loading="lazy"` here. Also use `.svg` format of icons here too.

Reply:
I fixed it